package.json

```diff
@@ -14,7 +14,6 @@
     "author": "BaM",
     "license": "ISC",
     "dependencies": {
-        "chart.js": "^4.4.0",
```

Comment:
So in production this package will be uninstalled automatically?

Reply:
Your production instance is destroyed and re-created every time you deploy, so it won't be literally uninstalled, it just won't be installed on the next deploy.

Reply:
Man, this videos are awesome. Tnx for making this settings page!